media/utils.py

```diff
@@ -54,9 +56,17 @@ def oembed_from_id(object_id, oembed_url, service, field, pattern):
             _("Could not validate %(field)s due to invalid response from %(service)s.")
             % locals()
         ) from e
+    
+    # Youtube asset doesn't have this attribute
```

Comment:
since this is only for soundcloud, might not belong in a shared utility. you can return full html in the metadata and then extract what you do below directly in the soundcloud class. thoughts?

Reply:
100% agreed